samples/samples.js

```diff
@@ -117,6 +117,18 @@
 			path: 'scales/time/combo.html'
 		}]
 	}, {
+		title: 'TimeSeries scale',
```

Comment:
A single example would be fine with me. I would just suggest that we not use the combo chart as the example in that case. We were also talking about following up afterwards in a separate PR with some improvements to the time samples. E.g. make the samples show more points by default and add a dropdown which lets you see samples with minutes, hours, weeks, months, etc. instead of just days